script/Ciel/Day 13/13_01_01_CIEL13_4_1.txt

```diff
@@ -156,12 +177,13 @@ The voice disappears completely.
 [sha:8dfe2b8e332df0b8c1a46b0043168dc0f75485d3]{
 -- Page 363, Offset 34300.
 -- かつん、という足音と、伸びてくる影。
-Along with the sound of her footsteps comes her looming shadow.
+The footsteps shuffle to a halt,
+ and a shadow falls across the floor.
 }
 [sha:b1006b14adbe2faa9fa8a1f00ab22805ff68f71b]{
 -- Page 363, Offset 34301.
 -- ―――代行者が、俺の前に現れた。
-―――The Executor appears before my eyes.
+―――The Executor has appeared before me.
 }
 [sha:16cd54b66d7cd1c56b2f2c0fee321bf8da674689]{
 -- Page 364, Offset 34302.
```

Comment:
Re: first, I guess my concern with that change is that it could make it feel like he was already fighting but not wholeheartedly or something? My interpretation is more like this is her saying "Oh good, you've finally decided to fight [instead of run away], now I can have a clean conscience about this"

Reply:
Well, that's kinda what's been happening, I guess? He was "fighting" whether he wanted to or not. What happened now was the he came to terms with that reality. Well, regardless... it's not really a big deal.